docs/docs/deploying-to-now.md

```diff
@@ -1,80 +1,31 @@
 ---
-title: Deploying to Now
+title: Deploying to Vercel Now
 ---
 
-[ZEIT Now](https://zeit.co/now) is a [cloud platform for serverless deployment](https://zeit.co/docs/v2/getting-started/introduction-to-now/) that you can use to deploy your Gatsby projects and [alias them](https://zeit.co/docs/v2/domains-and-aliases/aliasing-a-deployment/) to your personal domain or a free `.now.sh` suffixed URL.
+[Vercel Now](https://vercel.com/now) is a cloud platform for websites and serverless APIs, that you can use to deploy your Gatsby projects to your personal domain (or a free `.now.sh` suffixed URL).
 
 This guide will show you how to get started in a few quick steps:
 
-## Step 1: Getting Started with Gatsby
+## Step 1: Installing Now CLI
 
-If you haven't already [set up a Gatsby project](https://www.gatsbyjs.org/docs/quick-start) you can do so by first installing Gatsby globally:
-
-```shell
-npm install --global gatsby-cli
-```
-
-Then generate a project with the following command:
-
-```shell
-npx gatsby new <your project name>
```

Comment:
💡

Reply:
I believe that line is no longer a valid add because it's no longer present in the master branch, do you think it's something that I should add in notwithstanding?

Reply:
Neevrmind. Sorry for the mix up

Reply:
@Developerayo can you update your fork of gatsby? I can't seem to be able to merge master into it. Kindly do this on your end